modules/exploits/multi/http/wp_db_backup_rce.rb

```diff
@@ -0,0 +1,238 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
```

Comment:
This could probably be `ExcellentRanking` because of the command injection and reliable vector. I would add module traits to granularize.